packages/itwinui-react/src/core/Surface/Surface.tsx

```diff
@@ -46,25 +86,34 @@ export type SurfaceProps = {
  * <Surface>Surface Content</Surface>
  * <Surface elevation={2}>Surface Content</Surface>
  */
-export const Surface = React.forwardRef(
-  (props: SurfaceProps, ref: React.RefObject<HTMLDivElement>) => {
-    const { elevation, className, style, children, ...rest } = props;
-    useTheme();
+export const Surface = Object.assign(
+  React.forwardRef(
+    (props: SurfaceProps, ref: React.RefObject<HTMLDivElement>) => {
+      const { elevation, className, style, children, layout, padded, ...rest } =
+        props;
+      useTheme();
 
-    const _style = {
-      '--iui-surface-elevation': getSurfaceElevationValue(elevation),
-      ...style,
-    };
-    return (
-      <div
-        className={cx('iui-surface', className)}
-        style={_style}
-        ref={ref}
-        {...rest}
-      >
-        {children}
-      </div>
-    );
+      const _style = {
+        '--iui-surface-elevation': getSurfaceElevationValue(elevation),
+        ...style,
+      };
+      return (
+        <div
+          className={cx('iui-surface', className)}
+          style={_style}
+          ref={ref}
+          data-iui-layout={layout ? 'true' : undefined}
+          data-iui-padded={padded ? 'true' : undefined}
```

Comment:
I noticed in the css that `data-iui-padded` only affects `iui-surface-body`, but the demo pages have this data attribute applied to `iui-surface`. Should the data attribute just apply to `iui-surface-body`?

Reply:
Left a comment here: https://github.com/iTwin/iTwinUI/pull/1044/files#r1134461684